airbyte-integrations/connectors/source-google-ads/unit_tests/test_source.py

```diff
@@ -135,11 +139,10 @@ def test_read_missing_stream(config, mock_account_info):
             )
         ]
     )
-
-    try:
+    with pytest.raises(KeyError) as error:
         list(source.read(AirbyteLogger(), config=config, catalog=catalog))
-    except KeyError as error:
```

Comment:
why did you change this test? has the behavior of the source changed?

Reply:
Yes, the `KeyError` was not raised, so just corrected it to explicitly check the error message related to the test.